browse/templates/home/special-message.html

```diff
@@ -1,20 +1,21 @@
 {#- Special messages appear in a column to the side of the website intro/news section,
   each in their own column. Always apply the style "message-special" to the column.
   You can also apply the style "dark" to override the red styles and add a message with a black background instead.
-  There should preferably be just one, and never more than two. -#}
+  There should can be just one, never more than two. -#}
```

Comment:
You should can this message in favor of one with better grammar...

Reply:
It's just a comment ...